scheds/rust/scx_layered/src/bpf/intf.h

```diff
@@ -162,4 +163,8 @@ struct layer {
 	unsigned int		perf;
 };
 
+struct user_ctx {
```

Comment:
Given that we have uid matches, `user` may be a bit confusing? How about just `override_ctx` or `overrides`.